pkg/master/master.go

```diff
@@ -567,12 +567,13 @@ func (m *Master) init(c *Config) {
 
 		"podTemplates": podTemplateStorage,
 
-		"replicationControllers": controllerStorage,
-		"services":               service.NewStorage(m.serviceRegistry, m.endpointRegistry, serviceClusterIPAllocator, serviceNodePortAllocator),
-		"endpoints":              endpointsStorage,
-		"nodes":                  nodeStorage,
-		"nodes/status":           nodeStatusStorage,
-		"events":                 eventStorage,
+		"replicationControllers":        controllerStorage,
+		"replicationControllers/status": controllerStatusStorage,
```

Comment:
Case conversion happens automatically.  Internally we have camel case in
many parts of the code.

On Thursday, October 8, 2015, Tim Hockin <notifications@github.com> wrote:

> In pkg/master/master.go
> <https://github.com/kubernetes/kubernetes/pull/14672#discussion_r41474385>
> :
>
> > @@ -567,12 +567,13 @@ func (m *Master) init(c *Config) {
> >
> >  		"podTemplates": podTemplateStorage,
> >
> > -		"replicationControllers": controllerStorage,
> > -		"services":               service.NewStorage(m.serviceRegistry, m.endpointRegistry, serviceClusterIPAllocator, serviceNodePortAllocator),
> > -		"endpoints":              endpointsStorage,
> > -		"nodes":                  nodeStorage,
> > -		"nodes/status":           nodeStatusStorage,
> > -		"events":                 eventStorage,
> > +		"replicationControllers":        controllerStorage,
> > +		"replicationControllers/status": controllerStatusStorage,
>
> didn't we convert everything to all-lowercase?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/kubernetes/kubernetes/pull/14672/files#r41474385>.
>